src/server/config.rs

```diff
@@ -260,6 +260,17 @@ pub struct Config {
     // Interval to inspect the network latency between tikv and tikv for slow store detection.
     // If it set to 0, it will disable the inspection.
     pub inspect_network_interval: ReadableDuration,
+    /// Enable graceful shutdown for TiKV server.
+    /// When enabled, TiKV will perform graceful shutdown operations like
+    /// leader eviction before terminating.
+    #[online_config(skip)]
+    pub enable_graceful_shutdown: bool,
+
+    /// Timeout for leader eviction during graceful shutdown.
+    /// After this timeout, TiKV will proceed with shutdown even if
+    /// some regions haven't completed leader transfer.
+    #[online_config(skip)]
+    pub evict_leader_timeout: ReadableDuration,
```

Comment:
```suggestion
    pub graceful_shutdown_timeout: ReadableDuration,
```

How about `graceful_shutdown_timeout`? It's more intuitive.